lib/std/os/bits/linux.zig

```diff
@@ -951,7 +951,7 @@ pub fn cap_valid(u8: x) bool {
 }
 
 pub fn CAP_TO_MASK(cap: u8) u32 {
-    return @as(u32, 1) << u5(cap & 31);
+    return @as(u32, 1) << @truncate(u5, cap & 31);
```

Comment:
Ideally this would be `@as` instead of `@truncate`. Generally, `@as` is preferable because it does not have the ability to destroy information. However, we don't yet have enough Runtime Value Hints so that Zig knows that the result of `x & 31` can fit into a `u5`:

```zig
test "aoeu" {
    _ = CAP_TO_MASK;
}   
pub fn CAP_TO_MASK(cap: u8) u32 {
    return @as(u32, 1) << @as(u5, cap & 31);
}   
```

```
/home/andy/tmp/test.zig:6:39: error: expected type 'u5', found 'u8'
    return @as(u32, 1) << @as(u5, cap & 31);
                                      ^
/home/andy/tmp/test.zig:6:39: note: unsigned 5-bit int cannot represent all possible unsigned 8-bit values
    return @as(u32, 1) << @as(u5, cap & 31);
                                      ^
```

... which makes sense, otherwise we would not even need the `@as` cast.

The next best thing here would be `@intCast` since it asserts no information is destroyed.